lib/lsperfm.rb

```diff
@@ -1,21 +1,37 @@
 require 'yaml'
 require 'lsperfm/core'
+require 'terminal-table'
 
 module LogStash
   module PerformanceMeter
 
     extend self
 
     def invoke
-      debug   = !!ENV['DEBUG']
-      headers = !!ENV['HEADERS']
+      debug           = 'false' != ENV.fetch('DEBUG', 'false')
+      display_headers = 'false' != ENV.fetch('HEADERS', 'true')
+      table_output    = 'false' != ENV.fetch('TABLE_OUTPUT', 'false')
```

Comment:
I was surprised to learn the ENV hash didn't have a better structure to deal with true or false values, instead of by presence & stringifying everything - making !!'false' => true . This fixes that case.